telegram/ext/_application.py

```diff
@@ -948,7 +951,10 @@ def create_task(
         .. seealso:: :wiki:`Concurrency`
 
         Args:
-            coroutine (:term:`coroutine function`): The coroutine to run as task.
+            coroutine (:term:`awaitable`): The awaitable to run as task.
+
+            .. versionchanged:: 20.2
+                Accepts :class:`asyncio.Future` and generator-based coroutine functions.
```

Comment:
```suggestion
                .. versionchanged:: 20.2
                    Accepts :class:`asyncio.Future` and generator-based coroutine functions.
```

That fixes the failing doc test :)